packages/admin-ui/src/common/types.ts

```diff
@@ -429,13 +429,22 @@ export type ContainerState =
   | "exited" // exited A container that ran and completed("stopped" in other contexts, although a created container is technically also "stopped")
   | "dead"; // dead A container that the daemon tried and failed to stop(usually due to a busy device or resource used by the container)
 
-export type ChainDriver =
+export type ChainDriver = ChainDriverType | ChainDriverSpecs;
+
+export type ChainDriverSpecs = {
+  driver: ChainDriverType;
+  serviceName?: string;
+  portNumber?: number;
+};
+
+export type ChainDriverType =
   | "bitcoin"
   | "ethereum"
   | "ethereum2"
   | "ethereum2-beacon-chain-prysm"
```

Comment:
We should add another alias `beacon-chain` that's the correct name for the node, before `"ethereum2"`. Is any package right now using `"ethereum2"`? If not, keep only
```
  | "beacon-chain"
  | "ethereum2-beacon-chain-prysm"
```

Reply:
Use `ethereum-beacon-chain`